verilog/analysis/checkers/explicit_parameter_storage_type_rule.cc

```diff
@@ -100,7 +100,7 @@ void ExplicitParameterStorageTypeRule::HandleSymbol(
 // a common type that can't be handled well in some old tools.
 absl::Status ExplicitParameterStorageTypeRule::Configure(
     absl::string_view configuration) {
-  static const std::vector<absl::string_view> allowed = {"string"};
+  static const std::vector<absl::string_view> allowed = {"", "string"};
```

Comment:
this change and the one in the corresponding test don't like like they are meant to be in this PR.

Reply:
This change was a by-product of this change. The lint rule print generates the following statement:

`explicit-parameter-storage-type=exempt_type:`

which ExplicitParameterStorageTypeRule::Configure rejects. So I added the rules default value of an empty string to the allowed list.